src/_data/ghost.js

```diff
@@ -54,7 +94,7 @@ module.exports = async () => {
   const authors = [];
   const primaryAuthors = getUniqueList(
     posts.map(post => post.primary_author),
-    'id'
+    'slug'
   );
   primaryAuthors.forEach(author => {
     // Attach posts to their respective author
```

Comment:
Hmmm, that's strange. `primary_author` is a field from Ghost but as I dint want to modify this file a lot, I edited the data from Hashnode(the `process-batch.js` file inside `hashnode` folder) to match Ghost so that we can this file can then generate the pages easily.

Reply:
Fixed this. I had forgotten to set the `slug` field for authors when processing Hashnode posts. Thanks for catching this